plugins/X3DReader/X3DReader.py

```diff
@@ -0,0 +1,902 @@
+# Seva Alekseyev with National Institutes of Health, 2016
```

Comment:
Note that the code is also AGPL v3, as it's based on other AGPL v3 code (which infects this code)